cmake/BundledLibXml2.cmake

```diff
@@ -38,6 +38,10 @@ function(use_bundled_libxml2 SOURCE_DIR BINARY_DIR)
                 "-DCMAKE_INSTALL_PREFIX=${BINARY_DIR}/thirdparty/libxml2-install")
     endif()
 
+    if (CMAKE_VERSION VERSION_GREATER_EQUAL 3.24)
+        cmake_policy(SET CMP0135 OLD) # Restore the timestamps from the archive https://gitlab.kitware.com/cmake/cmake/-/issues/24003
+    endif()
```

Comment:
Yeah that was my original intent, but the latest cmake on centos7 doesnt have this feature, so that would only work if I duplicated the whole ExternalProject_Add into an `if (CMAKE_VERSION VERSION_GREATER_EQUAL 3.24)` statement. 

I also tried to put the "DOWNLOAD_EXTRACT_TIMESTAMP ON" and "" (based on cmakeversion) into a variable and pass that into the ExternalProject_Add but I wanst able to make that work.